supressions_cve.xml

```diff
@@ -77,4 +77,25 @@
         <sha1>ab0203590a39ad2fe4c225ddfad70e6a552c8445</sha1>
         <cpe>cpe:/a:bouncycastle:legion-of-the-bouncy-castle-fips-java-api</cpe>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: jackson-databind-2.13.5.jar (https://github.com/FasterXML/jackson-databind/issues/3973)
```

Comment:
I don't know if I understand why we shouldn't care. Maybe we should add to `notes` a better description?

Reply:
It was a false positive. CVE is revoked now so I'm closing this PR